client/ayon_traypublisher/plugins/publish/collect_editorial_instances.py

```diff
@@ -27,8 +27,11 @@ def process(self, instance):
             otio_timeline_string)
 
         instance.context.data["otioTimeline"] = otio_timeline
-        instance.context.data["editorialSourcePath"] = (
-            instance.data["editorialSourcePath"])
+
+        # Just for trimming from media in simple editorial workflow
+        if editorial_source_path := instance.data.get("editorialSourcePath"):
+            instance.context.data["editorialSourcePath"] = \
```

Comment:
Why is `editorialSourcePath` being set to `context.data`? I've tried to find where it's used, and it looks like it is not...

Reply:
look into `ayon-traypublisher\client\ayon_traypublisher\plugins\publish\extract_trim_video_audio.py`

Reply:
That is using `instance.data["editorialSourcePath"]`, not `context.data["editorialSourcePath"]`...

Reply:
yeah the logic could be actually improved. here are changes ecfd8989c3d723b321a75616e409db4a250188b8. But briefly, the `editorialSourcePath` key could be added directly to clip instance products rather then passing it via context from otio editorial instance to other instances.